build_tools/bazel_to_cmake/bazel_to_cmake_targets.py

```diff
@@ -25,15 +25,15 @@
         "dear_imgui::impl_sdl", "dear_imgui::impl_vulkan"
     ],
     # LLVM
-    "@llvm-project//llvm:asm_parser": ["LLVMAsmParser"],
-    "@llvm-project//llvm:ir": ["LLVMCore"],
-    "@llvm-project//llvm:execution_engine": ["LLVMExecutionEngine"],
-    "@llvm-project//llvm:passes": ["LLVMPasses"],
-    "@llvm-project//llvm:target_base": ["LLVMTarget"],
-    "@llvm-project//llvm:support": ["LLVMSupport"],
-    "@llvm-project//llvm:orcjit": ["LLVMOrcJIT"],
-    "@llvm-project//llvm:tablegen": ["LLVMTableGen"],
-    "@llvm-project//llvm:x86_target": ["LLVMX86CodeGen"],
+    "@llvm-project//llvm:AsmParser": ["LLVMAsmParser"],
+    "@llvm-project//llvm:Core": ["LLVMCore"],
+    "@llvm-project//llvm:ExecutionEngine": ["LLVMExecutionEngine"],
+    "@llvm-project//llvm:Passes": ["LLVMPasses"],
+    "@llvm-project//llvm:Target": ["LLVMTarget"],
+    "@llvm-project//llvm:Support": ["LLVMSupport"],
+    "@llvm-project//llvm:OrcJIT": ["LLVMOrcJIT"],
+    "@llvm-project//llvm:TableGen": ["LLVMTableGen"],
+    "@llvm-project//llvm:X86CodeGen": ["LLVMX86CodeGen"],
```

Comment:
@GMNGeoffrey With these changes, wouldn't it be better to implement an equivalent to `_convert_mlir_target(target)`?

Reply:
Uh, sorry, seems this wasn't merged into the master and just a test branch.

Reply:
This was actually merged as a separate PR. This was part of a massive refactor I was doing internally, so I didn't do anything special for IREE. I'll come back and create an equivalent mapping in bazel_to_cmake in a bit